vignettes/cfa-draft-acg.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Confirmatory Factor Analysis in MplusAutomation"
+author: 'Adam Garber, Joshua Wiley'
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = TRUE, message = FALSE, warning = FALSE)
+```
+
+### Load packages
+```{r}
+
+library(MplusAutomation)
+library(tidyverse);library(here);library(gt) # For tidiness & tables. 
+library(semPlot);library(DiagrammeR) # For creating path diagrams.
```

Comment:
Would it be possible to use tidySEM for graphing? I've developed it to work in tandem with MplusAutomation

Reply:
Yes, definitely! I will read up on your documentation as I haven't used these new functions yet.

Reply:
If I can help out, let me know :)